### PR TITLE
TECH-1771 - Deploying Staging app to Kubernetes as well

### PR DESCRIPTION
**Pull request by OleksandrUA**

### CODE REVIEW

**Review by jeannettemcd — APPROVED** [review]
LGTM

File: .github/workflows/aws-staging.yaml [hunk 1]
```diff
@@ -73,14 +73,46 @@ jobs:
...
+        ECR_REGISTRY: ${{ steps.login-ecr.outputs.registry }}
       run: |
-        aws ecs update-service --cluster $CLUSTER_NAME --service chainlog-ui-staging --force-new-deployment --region $AWS_REGION > /dev/null
-        aws ecs update-service --cluster $CLUSTER_NAME --service chainlog-logger-staging --force-new-deployment --region $AWS_REGION > /dev/null
+        sed -i 's/${ECR_REGISTRY}/'$(echo $ECR_REGISTRY)/ deploy/staging/chainlog-logger.yaml
+        sed -i 's/${ECR_REGISTRY}/'$(echo $ECR_REGISTRY)/ deploy/staging/chainlog-ui.yaml
 
-    - name: Wait for all services to become stable
-      uses: oryanmoshe/ecs-wait-action@v1.3
+    - name: Configure AWS credentials to connect to EKS cluster
```
Comment (eskp) [inline .github/workflows/aws-staging.yaml:84]: Already done on line 21 (but with different keys?)

File: .github/workflows/aws-prod.yaml [hunk 2]
```diff
@@ -12,17 +12,17 @@ jobs:
...
-      uses: actions/checkout@v2
+      uses: actions/checkout@v3
 
     - name: Configure AWS credentials
-      uses: aws-actions/configure-aws-credentials@v1
+      uses: aws-actions/configure-aws-credentials@v2
       with:
-        aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
-        aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        aws-access-key-id: ${{ secrets.EKS_AWS_ACCESS_KEY_ID }}
```
Comment (eskp) [inline .github/workflows/aws-prod.yaml:24]: I'd call these `PROD_AWS_ACCESS_KEY_ID` and `STAGING_AWS_ACCESS_KEY_ID` for prod and staging respectively. As they're used for more than EKS

**Review by eskp — CHANGES_REQUESTED** [review]

**Comment by eskp** [comment]
Can you combine this PR with https://github.com/makerdao/chainlog-ui/pull/16?

File: .github/workflows/aws-prod.yaml (hunk 2 above)
Comment (OleksandrUA) [reply .github/workflows/aws-prod.yaml:24]: @eskp done, I am using Github Actions environments here so secret names don't need to have STAGING_ or PROD_ prefixes

**Review by eskp — APPROVED** [review]

**Pull request merged**